### DISCUSSION TITLE
Improve track point list behaviour

### DISCUSSION THREAD

**Discussion by wega789** [osmandapp/OsmAnd#19986] [Ideas]
Tipping one waypoint on a track does not select that point in the point list of the track. This would allow to go to the previous or next point and enter its context menu. This is useful when waypoints are very close to each other. I described the use case in Q&A #19886. This seems to be not too difficult to implement, but there might be other approaches, e.g. additional items "Got to next/previous point" in the track point context menu.

**Comment by tommycrock** [comment]
Would be very helpful. At the moment a recorded track with thousands of points is essentially uneditable when often all I want to do is chop off the last few minutes of recording.
